Khasi/Khasi_song_lyrics

```diff
@@ -0,0 +1,33 @@
+Hoi Kiw
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

Reply:
I d prefer to edit my files in Linux vim editor. How do add a hyperlink there to redirect to my language file story?